### PR TITLE
cosmwasm: update verify script

### LINKED ISSUE #1878: Injective: Deployed on-chain bytecode verify script
It would be nice to have a script that verifies that the deployed on-chain bytecode matches the local object file

### PR DESCRIPTION
**Pull request by panoel**
This PR updates the cosmwasm verify script to handle terra, terra2, injective, and xpla

Fixes #1878

### CODE REVIEW

File: cosmwasm/verify [hunk 1]
```diff
@@ -3,20 +3,37 @@
 set -euo pipefail
 
 usage="Usage:
-  $(basename "$0") [-h] [-n network] <.wasm file> <code id> -- Verify that the deployed on-chain bytecode matches the local object file
+  $(basename "$0") [-h] [-n network] [-c chain] [-w .wasm file] [-i code id] -- Verify that the deployed on-chain bytecode matches the local object file
```
Comment (kcsongor) [inline cosmwasm/verify:6]: This script is referenced here: https://github.com/wormhole-foundation/wormhole/blob/551972d74da30fa7fee8cf016351089a03e82586/scripts/contract-upgrade-governance.sh#L449-L467 so we should update that script too.

Comment (panoel) [reply cosmwasm/verify]: Updated

File: cosmwasm/verify [hunk 2]
```diff
@@ -29,28 +50,105 @@ while getopts ':hn:' option; do
...
      echo "$usage" >&2
      exit 1
      ;;
 esac
 
-[ $# -ne 2 ] && { echo "$usage" >&2; exit 1; }
-obj_file=$1
-code_id=$2
+if [[ -z $url ]]; then
+  printf "The combination of $network and $chain is not supported.\n"
```
Comment (kcsongor) [inline cosmwasm/verify:129]: When printing errors, make sure to print them to stderr and not stdout
```suggestion
  printf "The combination of $network and $chain is not supported.\n" 1>&2 
```

File: cosmwasm/verify [hunk 3]
```diff
@@ -29,28 +50,105 @@ while getopts ':hn:' option; do
...
+  exit 1
+fi
+
+if [[ ! -z $contract_addr && -z $code_id_url ]]; then
+  printf "There is no contract to code_id conversion for the combination of $network and $chain.\n"
```
Comment (kcsongor) [inline cosmwasm/verify:134]:
```suggestion
  printf "There is no contract to code_id conversion for the combination of $network and $chain.\n" 1>&2 
```

File: cosmwasm/verify (hunk 2 above)
Comment (panoel) [reply cosmwasm/verify]: Fixed

File: cosmwasm/verify (hunk 3 above)
Comment (panoel) [reply cosmwasm/verify]: Fixed

**Review by jumpsiegel — APPROVED** [review]

**Review by kev1n-peters — APPROVED** [review]

**Pull request merged**